fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanSummary.java

```diff
@@ -38,6 +38,12 @@
 public class LoanSummary {
 
     // derived totals fields
+    @Column(name = "total_principal_derived", scale = 6, precision = 19)
+    private BigDecimal totalPrincipal;
+
+    @Column(name = "principal_through_capitalized_income_derived", scale = 6, precision = 19)
```

Comment:
`capitalized_income_derived` is enough... capitalizing income means turn it to principal, so implicitly we know that already...